test/jdk/java/net/httpclient/websocket/Abort.java

```diff
@@ -23,6 +23,7 @@
 
 /*
  * @test
+ * @modules java.net.http/jdk.internal.net.http.websocket
```

Comment:
Instead of adding this line to all tests, you can add a new `TEST.properties` that contains:
```properties
modules = java.net.http/jdk.internal.net.http.websocket
```
And only add `@modules` for files that require extra modules. See other `TEST.properties` in the JDK.

Reply:
Good call – this indeed reduces the size of touched files. Implemented in f5d347157bb8dd8e56457be489fc8899df286f5c.